src/lib/posts/how-to-review-a-pull-request.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'How to review a pull/merge request'
+date: '2024-05-08'
+updated: '2024-05-12'
+categories:
+    - 'coaching'
+    - 'dev'
+coverImage: '/images/kevin-ku-w7ZyuGYNpRQ-unsplash.jpg'
+excerpt: Why reviews matter, what to do, and some tips for etiquette
+---
+
+<script>
+    import Callout from "$lib/components/Callout.svelte";
+</script>
+
+Good reviews are a gift to your future self and teammates who have to maintain and understand the code being merged to ship new code on top of it.
+Since most developers spend more time reading code than writing it, practicing reading and reviewing code is has multiplicative benefits for you and eventually you'll find yourself reviewing code as written here right in your editor. 
```

Comment:
```suggestion
Since most developers spend more time reading code than writing it, practicing reading and reviewing code is has multiplicative benefits and soon you'll find yourself reviewing code as written here right in your editor. 
```